.gitlab-ci.yml

```diff
@@ -81,3 +83,28 @@ fire-caos-itest:
   script: gradle clean :submission:submission-webapp:itest -PenableFire=true --rerun-tasks --stacktrace
   variables:
     ITEST_FAIL_FACTOR: "8"
+
+build-fat-jar:
```

Comment:
looking good lets make the following changes:

1. Lets create an specialized version docker.yml file where we put this pipelines.
2. Lets change this pipeline image name to be `submitter:$CI_COMMIT_SHA-SNAPSHOT`
3. Lets create an job exactly as build-push-docker-image but only executed on prod (check other pipelines on example) that publish image but without SNAPSHOT

Reply:
changes done!